### DISCUSSION TITLE
UI navigation speed question for applications

### DISCUSSION THREAD

**Discussion by jvilhuber** [argoproj/argo-cd#7098] [Ideas]
When navigating through multiple apps, the UI seems to force me to go:
1. applications/
2. -> applications/applicationA
3. <- applications/
4. -> applications/applicationB

Is there a fast way to go directly from applicationA to applicationB?

In our deployments, with loads of apps, and with a slow network (I'm in SE Asia, VPN back to the home office in NorthAmerica), each loading of application/ takes somewhere around 5-10 seconds on a GOOD day (usually much slower).

I've been hoping for some '_fast application switcher_' in the argocd header (common to all pages, so available from application/applicationA), to help with this. It would be similar to the search-box on the main applications/ page, and selecting an app from the search takes you directly to that app, bypassing the main page.

Any tips or tricks I'm missing?
Regards,
jan

**Comment by todaywasawesome** [comment]
You can filter all the applications down to the set you want to look at on the main page and then do ctrl-click to open each one in a new window. 

It might be worth [making an issue](https://github.com/argoproj/argo-cd/issues/new/choose). [#4700](https://github.com/argoproj/argo-cd/issues/4700) made it so you can use `/` to jump to the search but that doesn't work on the application view which would be a cool feature.

**Reply by jvilhuber** [reply]
Filed an issue. Thanks.
https://github.com/argoproj/argo-cd/issues/7186
